index.js

```diff
@@ -507,11 +507,23 @@ class NewRelic {
   }
 
   sendConsole(type, args) {
-    const argsStr = JSON.stringify(args);
+
+    // Remove circular structures from the args so we can convert to JSON
```

Comment:
If this doesn't one causes any important data losses in the NR web console, I'm happy with it

Reply:
@kthai-nr I guess If Crockford's [decycle](https://github.com/douglascrockford/JSON-js/blob/master/cycle.js) is used, cyclic references can also be preserved.

Reply:
I don't expect data loss but Crockford's decycle was mentioned in the MDN docs. I'll take a deeper look into it before our next release and do some testing against it. Thanks for the feedback!